### DISCUSSION TITLE
egui_plot::item::Text --> how to define the font-size?

### DISCUSSION THREAD

**Discussion by jjimenezroda** [emilk/egui#4470] [Q&A]
I'm plotting some text to some egui_plot using the following code snippet

```rust
.show(ui, |plot_ui|{
                plot_ui.points(state_changes_markers.name("State changes"));

                //Plot state names
                for state_change in data_set2.state_changes.iter(){
                    plot_ui.text(Text::new(PlotPoint::new(state_change.0.timestamp() as f64, state_change.1 as f64 + (1.5 * zoom_factor as f64)), state_change.2.clone()));
                }
```

Does anyone know how can I set the font_size of the text? I would like to change it based on the zoom factor of the plot itself.

Thanks in advance!

**Comment by YgorSouza** [comment]
Put the string in a [RichText](https://docs.rs/egui/0.27.2/egui/widget_text/struct.RichText.html), then you can set the size, color, font etc.

See also #3277, which I think is what you are trying to do.
